lib/ts/controllers/s-expandable-control.ts

```diff
@@ -141,17 +156,17 @@
                     }
                 }
             }
-        },
+        };
 
-        disconnect: function () {
-            this.events.forEach(function (e) {
-                this.element.removeEventListener(e, this.listener);
+        disconnect() {
+            this.events.forEach(e => {
+                this.element.removeEventListener(e, this.listener as EventListener);
```

Comment:
Just use a cast here, won’t it always be an `EventListener`?

Reply:
`x as Foo` and `<Foo>x` mean exactly the same thing in TS (type erasure)

Reply:
Updated `listener(e: Event)` to match the `EventListener` interface so a cast is no longer needed.